src/Main.java

```diff
@@ -1,5 +1,223 @@
+                arrayOfEvenNumbers[j] = arrayOfNumbers[i];
+                j++;
+            }
+        }
+
+        System.out.println("Even numbers for range " + lowerNumber + " to " + largerNumber + " are: ");
+
+        for (int i = 0; i <= arrayOfEvenNumbers.length - 1; i++) {
+            System.out.print(arrayOfEvenNumbers[i] + ",");
+        }
+
+        // Zadatak 6
+
+        System.out.println("Task 6: ");
+
+        int[] arrayOne = {1, 2, 3};
+        int[] arrayTwo = {4, 5, 6, 7};
+        int[] arrayThree = {7, 8};
+        int lengthFinal = arrayOne.length + arrayTwo.length + arrayThree.length;
+        int[] arrayOneTwoThree = new int[lengthFinal];
+
+        for (int i = 0; i <= arrayOne.length - 1; i++) {
+            arrayOneTwoThree[i] = arrayOne[i];
+        }
+        for (int i = 0; i <= arrayTwo.length - 1; i++) {
+            arrayOneTwoThree[i + (arrayOne.length)] = arrayTwo[i];
+        }
+        for (int i = 0; i <= arrayThree.length - 1; i++) {
+            arrayOneTwoThree[(arrayOne.length + arrayTwo.length) + i] = arrayThree[i];
+        }
+
+        for (int i = 0; i <= arrayOneTwoThree.length - 1; i++) {
+            System.out.println(arrayOneTwoThree[i]);
+        }
+
+        // Zadatak 7
+
+        System.out.println("Task 7: ");
+
+        int[] arrayFirst = {1, 2, 3, 4, 5, 6, 7};
+        int[] arraySecond = {1, 1, 1, 1, 1};
+        int maxValue = max(arrayFirst.length, arraySecond.length);
+        int minValue = min(arrayFirst.length, arraySecond.length);
+        int[] finalArray = new int[maxValue];
+
+        for (int i = 0; i <= minValue - 1; i++) {
+            finalArray[i] = arrayFirst[i] + arraySecond[i];
+        }
+        if (arrayFirst.length > arraySecond.length) {
+            for (int i = arraySecond.length; i <= finalArray.length - 1; i++) {
+                finalArray[i] = arrayFirst[i];
+            }
+        } else {
+            for (int i = arrayFirst.length; i <= finalArray.length - 1; i++) {
+                finalArray[i] = arraySecond[i];
+            }
+        }
+        for (int i = 0; i <= finalArray.length - 1; i++) {
+            System.out.println(finalArray[i]);
+        }
+
+        // Zadatak 8
+
+        System.out.println("Task 8: ");
+
+        String[] arrayOfString = {"Ima", "trp", "li", "u", "ART", "slm", "stringu", "znak", "krk", "brb"};
+        String[] arrayOfStringToUpper = new String[arrayOfString.length];
+        System.out.println(arrayOfString.length);
+        int numberOfStringsToRemove = 0;
+
+        for (int i = 0; i <= arrayOfString.length - 1; i++) {
+            arrayOfStringToUpper[i] = arrayOfString[i].toUpperCase();
+            if (arrayOfStringToUpper[i].contains("A") || arrayOfStringToUpper[i].contains("E") || arrayOfStringToUpper[i].contains("I") || arrayOfStringToUpper[i].contains("O") || arrayOfStringToUpper[i].contains("U")) {
+                numberOfStringsToRemove = numberOfStringsToRemove + 1;
```

Comment:
since you are tagging the number of strings to be remove, you also could have saved the indexes of them, so that you avoid the contains logic again later